src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```

Comment:
```suggestion
    private static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```
if we use this variable only in this class make it private

Reply:
done